### DISCUSSION TITLE
Downgrading subscription

### DISCUSSION THREAD

**Discussion by Localyze** [community/community#22073] [New to GitHub]
<p>I’ve been trying to end my subscription charge, as I don’t require github service any longer, but no matter what I try I can’t figure it out. Please help.</p>

**Comment by canuckjacq** [comment]
<p>:wave: Welcome!</p>
<p>Sorry you’re having this trouble. We have a guide here in case that might help:</p>
<p><a href="https://help.github.com/github/setting-up-and-managing-billing-and-payments-on-github/downgrading-your-github-subscription#downgrading-your-user-accounts-subscription" class="onebox" target="_blank">https://help.github.com/github/setting-up-and-managing-billing-and-payments-on-github/downgrading-your-github-subscription#downgrading-your-user-accounts-subscription</a></p>

**Comment by Localyze** [comment]
<p>I’ve read that before. I access the account via mobile device, and, try as I may, i cannot find anything in the top right corner, which accesses settings</p>

**Comment by Localyze** [comment, marked as answer]
<p>Nm I see it now… ty for the help</p>

**Comment by Dcos70** [comment]
Hi folks, I'm down grading my subscription momentarily while I'm studying. I'm actually learning how to become a dev and I shall return with knowledge..p.s  I like how this site is set up and do want to be part of it.
